### PR TITLE
Add admin ability to remove beatmapset covers

### PR DESCRIPTION
**Pull request by nekodex**
as it says on the tin

### CODE REVIEW

File: resources/lang/en/layout.php [hunk 1]
```diff
@@ -146,9 +146,9 @@
             'logs-index' => 'log',
             'beatmapsets' => [
                 '_' => 'beatmapsets',
-                'covers' => 'covers',
                 'show' => 'detail',
             ],
+            'beatmapsets-covers' => 'beatmapset covers',
```
Comment (nanaya) [inline resources/lang/en/layout.php]: keys with string values are before arrays.

File: routes/web.php [hunk 2]
```diff
@@ -20,6 +20,7 @@
 Route::group(['as' => 'admin.', 'prefix' => 'admin', 'namespace' => 'Admin'], function () {
     Route::get('/beatmapsets/{beatmapset}/covers', 'BeatmapsetsController@covers')->name('beatmapsets.covers');
     Route::post('/beatmapsets/{beatmapset}/covers/regenerate', 'BeatmapsetsController@regenerateCovers')->name('beatmapsets.covers.regenerate');
+    Route::post('/beatmapsets/{beatmapset}/covers/remove', 'BeatmapsetsController@removeCovers')->name('beatmapsets.covers.remove');
```
Comment (nanaya) [inline routes/web.php]: `Route::delete`?

File: resources/views/admin/beatmapsets/cover.blade.php [hunk 3]
```diff
@@ -18,20 +18,29 @@
...
+            <h2>{{$beatmapset->title}} - {{$beatmapset->artist}}</h2>
             <br>
-        @endforeach
+            {!! Form::open([
+                'route' => ['admin.beatmapsets.covers.regenerate', $beatmapset->beatmapset_id],
+                'method' => 'POST'
+            ]) !!}
+                <button>{{trans('admin.beatmapsets.covers.regenerate')}}</button>
+            {!! Form::close() !!}
+            {!! Form::open([
```
Comment (nanaya) [inline resources/views/admin/beatmapsets/cover.blade.php]: There's this `data-remote` thingy which can have `data-reload-on-success`.

[Example](/ppy/osu-web/blob/aa817fd71a1457055eec711e11af2ea6b80d8148/resources/views/beatmapset_watches/index.blade.php#L86).

File: routes/web.php (hunk 2 above)
Comment (nekodex) [reply routes/web.php]: idk, delete feels wrong in this case as it's not a singular resource that's being deleted?

File: app/Http/Controllers/Admin/BeatmapsetsController.php [hunk 4]
```diff
@@ -35,6 +36,14 @@ public function covers($id)
         return view('admin.beatmapsets.cover', compact('beatmapset'));
     }
 
+    public function removeCovers($id)
+    {
+        $beatmapset = Beatmapset::findOrFail($id);
+        $beatmapset->removeCovers();
+
+        return back();
```
Comment (nanaya) [inline app/Http/Controllers/Admin/BeatmapsetsController.php]: Causes one extra page request =D

**Pull request merged**